leits/MeetingBar
# feat(create-meeting): custom url for new meetings

### Status
**READY**

### Description
The PR adds the possibility for the user to use a custom url for new meetings.
The link is opened in the standard browser
If the link does not contain a http or https at the beginning, we will add the prefix.
If the link is empty or not valid, an alert is shown to the user.

### Steps to Test or Reproduce
Go to preferences, services and choose custom url as option for create meeting.
Add a valid url and try to create the meeting via menubar or shortcut.
The link should be opened in the standard browser.

Delete the url from the preference and try to create the meeting.
An alert should be displayed with a hint.

Enter a not valid url in the preference and try to create a meeting.
An alert should be displayed with a hint.
